logging_handler.go

```diff
@@ -0,0 +1,25 @@
+package msgraphgocore
```

Comment:
this should be part of kiota http and not just msgraph core as it could benefit people building clients for other APIs than MS Graph.

Reply:
Does this mean I duplicate the effort in both repos? Or I just move this to Kiota?

Reply:
just move it